Dockerfile

```diff
@@ -1,5 +1,5 @@
 FROM opensuse:tumbleweed
-MAINTAINER Fabian Neuschmidt fabian@neuschmidt.de
+LABEL maintainer="Fabian Neuschmidt fabian@neuschmidt.de"
```

Comment:
There have to some way to get the things directly by viewing it, here you did't give any difference between `persons` mail and his `name`, that's why we use `<>`.
Simply replace `"Fabian Neuschmidt fabian@neuschmidt.de"` to `"Fabian Neuschmidt <fabian@neuschmidt.de>"` will be more helpful, although your changes will work totally fine as we are not bounded here, we can set any `key` and his `value`.